tests/README.md

```diff
@@ -514,6 +515,23 @@ If you would like to test all profile's rules against their test scenarios:
 ./test_suite.py combined --libvirt qemu:///system ssg-test-suite-rhel8 --datastream ../build/ssg-rhel8-ds.xml ospp
 ```
 
+## Template-based testing
+
+```
+./test_suite.py template ... <template_name other_template>
```

Comment:
```suggestion
./test_suite.py template ... <template_name1>[ <template_name2> <template_name3> ...]
```

Maybe using something like this can make it clearer that it accepts multiple template_names, if I understood it correctly